### ISSUE TITLE
Not Identify RMv2 terminal word wraping M115 after "Rostoc"

### ISSUE THREAD

**Issue by T3CHKOMMIE** [geneb/OctoPrint-Delta-Calibration#6]
I think I found the bug that was causing this plugin to not detect i had a rostock seemecnc printer.

check this out:
Send: N22 M115*22Recv: okRecv: FIRMWARE_NAME:Repetier_0.92.2 FIRMWARE_DATE:20170109 MACHINE_TYPE:Rostock Max v2

your code is going through line by line, and my terminal is is next lining after Rostoc\nk Max v2

I changed the code to  if (line.includes("Rost") ||
                        line.includes("White Max v2.75")) {

and the script is identifying my max v2 correctly now. 

Not sure why the terminal is wrapping that line around but that is DEFINITELY the reason why some of our machines which are RM arent being identified. 

I hope this help. Ping me if you have questions. 

-Jules

**Comment by geneb** [comment]
Jules, if you want to submit a pull request for this change, I'll accept it.  Thanks for looking in to it!

**Comment by T3CHKOMMIE** [comment]
Hey Gene, my fix no longer appears to be working.... I am not sure what the issue is at this point. M115 shows Rostock Max v2 and im running newer repetier firmware. Tshooting the JS in octoprint is making my eyes bleed.

help?

**Comment by T3CHKOMMIE** [comment]
i had a js.bak files in the same directory for reverting changes. I wonder if octoprint was doing soemthing with it. after deleting that file it seems to be working now. ill keep testing.

**Comment by liranmas** [comment]
@T3CHKOMMIE the plugin does not identify my RMV2 as well, is it working for you as of now?

**Comment by T3CHKOMMIE** [comment]
I found it's a timing issue between firmware and plugin. If I disconnect my
octoprint from my printer and reconnect the plugin is loaded enough to ID
printer as RMv2. No issues now that I don't have my octoprint auto connect

**Comment by liranmas** [comment]
just to make sure, this plugin must have the HE280 hotend or the SE300 in order to work right? without the special effector plate it won't work?
The reason I'm asking is because MatterControl has a leveling feature that let's you measure each point and then it (I guess) gCode correcting the leveling

**Comment by T3CHKOMMIE** [comment]
Yes, that is correct. This plugin required the auto leveling hot end. I
have an he280.
